assets/app/view/game/sell_shares.rb

```diff
@@ -46,7 +46,10 @@ def render
           end
         end
 
-        h(:div, buttons.compact)
+        buttons = buttons.compact
+        h(:div, buttons) if buttons.any?
```

Comment:
are you missing a return here? 

did you test this code?

Reply:
My savior!

Reply:
I fixed this in both places where I added the explicit nil return. However, if you'd prefer me to hold on to this one until we've got more UI tests, I understand. I have verified buy/sell/par buttons but not the "choose"